server.js

```diff
@@ -16,21 +16,23 @@ var EdgesSchema = require('./src/schemas/EdgesSchema');
 var EdgesResolver = require('./src/resolvers/Edges');
 var SettingsResolver = require('./src/resolvers/Settings');
 var SettingsSchema = require('./src/schemas/SettingsSchema');
+var TopicsSchema = require('./src/schemas/TopicsSchema');
+var TopicsResolver = require('./src/resolvers/Topics');
 
 var app = express();
 
 app.use(function(req, res, next) {
-    res.header('Access-Control-Allow-Origin', '*');
-    res.header('Access-Control-Allow-Credentials', true);
-    res.header('Access-Control-Allow-Methods', 'OPTIONS,GET,POST,PUT,DELETE');
-    res.header('Access-Control-Allow-Headers', 'Content-Type, Authorization, x-ms-meta-data*,x-ms-meta-target*,x-ms-meta-abc');
+  res.header('Access-Control-Allow-Origin', '*');
```

Comment:
FYI: There's a really useful argument to `git add` with is `-p`. This will step you through all your changes one at a time and ask you if you want to stage them for commit. In this way, it's relatively easy to make targeted/one-purpose commits. In this specific case, you could have staged/committed all the lint changes first, pushed to a branch to make a PR, get through that PR (easy: it's all just whitespace changes so it'll take 5 mins to review and sign off) and then submit another PR for the functional change. Generally speaking, it's much easier to review code if it's only doing one thing: refactoring or adding functionality. If multiple purposes are mixed, it increases the mental tax on the reviewer's part as one has to be much more focused to not gloss over important parts.